lib/computer.rb

```diff
@@ -1,15 +1,50 @@
-require "./spec/spec_helper"
-
 class Computer
-  attr_reader :mark
+  attr_reader :mark, :opponent
 
-  def initialize(mark)
-    @moves = computer_player
+  def initialize(mark, board)
     @mark = mark
+    @board = board
   end
 
   def get_move
-    @moves.shift
-    # return a random available move
+    analyse_board
+  end
+
+  def opponent_mark
+    if @mark == "O"
```

Comment:
I have a hunch these two strings are duplicated elsewhere. If so, how can you remove that duplication?

Reply:
Resolved #33